Makefile

```diff
@@ -57,7 +57,8 @@ lint_ruby:
 
 .PHONY: lint_posix_newlines
 lint_posix_newlines:
-	git ls-files | grep -v vendor/ | xargs ./scripts/test_posix_newline.sh
+	# for some reason `git ls-files` is including 'manifests/cf-deployment' in its output...which is a directory
```

Comment:
Having this comment here causes make to print it on each run. Prefixing the comment with an `@` prevents this.

Reply:
Good spot. I've added the `@` and squashed it into the same commit.